### PR TITLE
Wallet (various fixes)

### PR DESCRIPTION
**Pull request by Mingling94**
CSS for wallet was off, didn't notice before last PR.

Boxes are now half the range of the height now and colored differently.

### CODE REVIEW

**Comment by Mingling94** [comment]
@DavidVorick boop, this one's probably more important than the other PR

File: app/plugins/Wallet/js/wallet.js [hunk 1]
```diff
@@ -115,17 +127,10 @@ function update() {
 			return;
 		}
 		
+		console.log('updating...');
```
Comment (lukechampine) [inline app/plugins/Wallet/js/wallet.js]: debugging output?

**Comment by lukechampine** [comment]
a lot of the CSS indentation seems off

**Comment by Mingling94** [comment]
Blast! I should just set tabbing to be something weird like 6 characters so its obvious when I'm actually looking at 4 spaces

**Comment by Mingling94** [comment]
@lukechampine Applied changes to your comments ^_^

**Pull request merged**